--- src/init2.c.orig
+++ src/init2.c
@@ -101,140 +101,22 @@
 	
 	/*** Prepare the "path" ***/
 	
-	/* Hack -- save the main directory */
-	ANGBAND_DIR = string_make(path);
-	
-	/* Prepare to append to the Base Path */
-	tail = path + strlen(path);
-	
-	
-#ifdef VM
-	
-	
-	/*** Use "flat" paths with VM/ESA ***/
-	
-	/* Use "blank" path names */
-	ANGBAND_DIR_APEX = string_make("");
-	ANGBAND_DIR_BONE = string_make("");
-	ANGBAND_DIR_DATA = string_make("");
-	ANGBAND_DIR_EDIT = string_make("");
-	ANGBAND_DIR_FILE = string_make("");
-	ANGBAND_DIR_HELP = string_make("");
-	ANGBAND_DIR_INFO = string_make("");
-	ANGBAND_DIR_SAVE = string_make("");
-	ANGBAND_DIR_PREF = string_make("");
-	ANGBAND_DIR_USER = string_make("");
-	ANGBAND_DIR_XTRA = string_make("");
-	ANGBAND_DIR_SCRIPT = string_make("");
-	
-	
-#else /* VM */
+	/* unused */
+	ANGBAND_DIR = string_make("");
 	
+	ANGBAND_DIR_APEX = string_make("/var/games/sil/apex");
+	ANGBAND_DIR_BONE = string_make("/var/games/sil/bone"); // N/A
+	ANGBAND_DIR_DATA = string_make("/var/games/sil/data");
+	ANGBAND_DIR_SAVE = string_make("/var/games/sil/save");
+	ANGBAND_DIR_USER = string_make("/var/games/sil/user");
 
-	/*** Build the sub-directory names ***/
-	
-	/* Build a path name */
-	strcpy(tail, "data");
-	ANGBAND_DIR_DATA = string_make(path);
-	
-	/* Build a path name */
-	strcpy(tail, "edit");
-	ANGBAND_DIR_EDIT = string_make(path);
-	
-	/* Build a path name */
-	strcpy(tail, "pref");
-	ANGBAND_DIR_PREF = string_make(path);
-	
-#ifdef PRIVATE_USER_PATH
-	
-	/* Build the path to the user specific directory */
-	path_build(buf, sizeof(buf), PRIVATE_USER_PATH, VERSION_NAME);
-	
-	/* Build a relative path name */
-	ANGBAND_DIR_USER = string_make(buf);
-	
-#else /* PRIVATE_USER_PATH */
-	
-	/* Build a path name */
-	strcpy(tail, "user");
-	ANGBAND_DIR_USER = string_make(path);
-	
-#endif /* PRIVATE_USER_PATH */
-	
-#ifdef USE_PRIVATE_SAVE_PATH
-	
-	/* Build the path to the user specific sub-directory */
-	path_build(buf, sizeof(buf), ANGBAND_DIR_USER, "scores");
-	
-	/* Build a relative path name */
-	ANGBAND_DIR_APEX = string_make(buf);
-	
-	/* Build the path to the user specific sub-directory */
-	path_build(buf, sizeof(buf), ANGBAND_DIR_USER, "save");
-	
-	/* Build a relative path name */
-	ANGBAND_DIR_SAVE = string_make(buf);
-	
-#else /* USE_PRIVATE_SAVE_PATH */
-	
-	/* Build a path name */
-	strcpy(tail, "apex");
-	ANGBAND_DIR_APEX = string_make(path);
-	
-	/* Build a path name */
-	strcpy(tail, "save");
-	ANGBAND_DIR_SAVE = string_make(path);
-	
-#endif /* USE_PRIVATE_SAVE_PATH */
-	
-	/* Build a path name */
-	strcpy(tail, "xtra");
-	ANGBAND_DIR_XTRA = string_make(path);
-	
-	/* Build a path name */
-	strcpy(tail, "script");
-	ANGBAND_DIR_SCRIPT = string_make(path);
-	
-#endif /* VM */
-	
-	
-#ifdef NeXT
-	
-	/* Allow "fat binary" usage with NeXT */
-	if (TRUE)
-	{
-		cptr next = NULL;
-		
-# if defined(m68k)
-		next = "m68k";
-# endif
-		
-# if defined(i386)
-		next = "i386";
-# endif
-		
-# if defined(sparc)
-		next = "sparc";
-# endif
-		
-# if defined(hppa)
-		next = "hppa";
-# endif
-		
-		/* Use special directory */
-		if (next)
-		{
-			/* Forget the old path name */
-			string_free(ANGBAND_DIR_DATA);
-			
-			/* Build a new path name */
-			sprintf(tail, "data-%s", next);
-			ANGBAND_DIR_DATA = string_make(path);
-		}
-	}
-	
-#endif /* NeXT */
-	
+	ANGBAND_DIR_EDIT = string_make("/usr/share/games/sil/edit");
+	ANGBAND_DIR_FILE = string_make("/usr/share/games/sil/file"); // N/A
+	ANGBAND_DIR_HELP = string_make("/usr/share/games/sil/help"); // N/A
+	ANGBAND_DIR_INFO = string_make("/usr/share/games/sil/info"); // N/A
+	ANGBAND_DIR_PREF = string_make("/usr/share/games/sil/pref");
+	ANGBAND_DIR_SCRIPT = string_make("/usr/share/games/sil/script");// N/A
+	ANGBAND_DIR_XTRA = string_make("/usr/share/games/sil/xtra");
 }
 
 
